src/rlx_assemble.erl

```diff
@@ -756,29 +756,39 @@ create_RELEASES(State, OutputDir, ReleaseFile) ->
             %% remove the RELEASES file in case it exists from a previous release build
             _ = rlx_file_utils:remove(filename:join([OutputDir, "releases", "RELEASES"]));
         _ ->
-            {ok, OldCWD} = file:get_cwd(),
-            try
-                %% set cwd to the outputdir so that the paths used for applications
-                %% in RELEASES start with ./lib/ and not the path of the output dir
-                %% which is a development path only.
-                ok = file:set_cwd(OutputDir),
-                case release_handler:create_RELEASES("./",
-                                                     "releases",
-                                                     ReleaseFile,
-                                                     []) of
-                    ok ->
-                        ok;
-                    {error, _Reason} ->
-                        ?log_debug("*WARNING* Creating RELEASES file failed. "
-                                   "The RELEASES file is required before release_handler can be used "
-                                   "to install a release."),
-                        ok
-                end
-            after
-                file:set_cwd(OldCWD)
+            case create_RELEASES(OutputDir, Release) of
+                ok ->
+                    ok;
+                {error, Reason} ->
+                    ?log_debug("*WARNING* Creating RELEASES file failed for reason: ~p "
+                               "The RELEASES file is required before release_handler can be used "
+                               "to install a release.", [Reason]),
+                    ok
             end
     end.
 
+create_RELEASES(OutputDir, Release) ->
+    RelName = atom_to_list(rlx_release:name(Release)),
+    RelVsn = rlx_release:vsn(Release),
+    ErtsVsn = rlx_release:erts(Release),
+    Apps = [{Name, Vsn, filename:join("./lib", [Name, "-", Vsn])}
+            || #{name := Name,
+                 vsn := Vsn,
+                 dir := _Dir} <- rlx_release:applications(Release)],
+
+    do_write_release(filename:join(OutputDir, "releases"),
+                     {release, RelName, RelVsn, ErtsVsn, Apps, permanent}).
+
+do_write_release(Dir, Release) ->
+    case file:open(filename:join(Dir, "RELEASES"), [write,{encoding,utf8}]) of
+        {ok, Fd} ->
+            ok = io:format(Fd, "%% ~s~n~tp.~n",
```

Comment:
since the first is encoded to utf8 you might want `~ts` instead of `~s`

Reply:
Hm, yea, I had just copied this directly from release_handler. One sec.